src/Class/Mur.java

```diff
@@ -35,4 +39,50 @@ public String toString(){
 		return res;
 	}
 
+	@Override
+	/**
+	 * Le isMoveOK doit être testé avant
+	 */
+	public boolean move(Coordonnees finalCoord){
+		//Set wall vertical ou horizontal
+		if(isWallBeHorizontal(finalCoord)) {
+			this.isHorizontal = true;
+		} else {
+			this.isHorizontal = false;
+		}
+		//Position du mur OK
+		this.setCoordonnees(finalCoord);
+		return true;
+	}
+
+	/**
+	 * Renvoi true si le mur posé à cette case est Horizontal
+	 * Renvoi false si le mur est vertical ou impossible
+	 * @param wallCoord
+	 * @return
+	 */
+	public static boolean isWallBeHorizontal(Coordonnees wallCoord){
+		if (wallCoord.getX() % 2 == 1 && wallCoord.getX()> 0 && wallCoord.getX()< 16 && wallCoord.getY() % 2 == 0 && wallCoord.getY()>= 0 && wallCoord.getY()< 15) { //Verif que la case est une case horizontale
+			return true;
+		} else {
+			return false;
+		}
+
+	}
+
+	/**
+	 * Renvoi true si le mur posé à cette case est Vertical
+	 * Renvoi false si le mur est horizontal ou impossible
+	 * @param wallCoord
+	 * @return
+	 */
+	public static boolean isWallBeVertical(Coordonnees wallCoord){
```

Comment:
cf commentaire sur le fait de n'avoir qu'une seule fonction

Reply:
cf commentaire précédent

Reply:
Après vérification, la fonction peut être supprimée si on ajoute une fonction. isCroisement ou is littleWall(), ce qui revient au même.